sdks/java/io/amazon-web-services/src/test/java/org/apache/beam/sdk/io/aws/s3/S3WritableByteChannelTest.java

```diff
@@ -60,29 +59,29 @@
 
   @Test
   public void write() throws IOException {
-    writeFromOptions(s3Options());
```

Comment:
Should it still support write from options?

Reply:
I've rewritten the test as much as possible to support that, but in the end the constructor takes a config instead of an options, and I think that's what we want.